src/components/HornedBeast.js

```diff
@@ -1,52 +1,77 @@
+import "bootstrap/dist/css/bootstrap.min.css";
 import React, { Component } from "react";
-import { Col, Container, Row } from "react-bootstrap";
+import { Button, Card } from "react-bootstrap";
 import jsonGallery from "../data/data.json";
-
-export default class HornedGallery extends Component {
+import SelectedBeast from "./SelectedBeast";
+export default class HornedBeast extends Component {
   constructor(props) {
     super(props);
     this.state = {
       favorites: 0,
+      shouldShowModal: false,
+      modalImageSrc: "",
+      imageDescription: "",
+      title: "",
     };
   }
 
+  // reference https://linguinecode.com/post/how-to-pass-parameter-values-to-onclick-react-function
+  openModal = (title) => (props) => {
+    this.setState({ shouldShowModal: true });
+    this.setState({ modalImageSrc: props.target.src });
+    this.setState({ imageDescription: props.target.alt });
+
+    this.setState({ title: title });
+    console.log(title);
+  };
+
+  closeModal = () => {
+    this.setState({ shouldShowModal: false });
+    this.setState({ modalImageSrc: "" });
+    this.setState({ imageDescription: "" });
+    this.setState({ imageTitle: "" });
```

Comment:
Great work on lab!

Fun tip here: These can all be in one setState call since it is an object. 

```
this.setState({ shouldShowModal: false });
    this.setState(
         { modalImageSrc: "" },
        { imageDescription: "" },
        { imageTitle: "" },
   );

Reply:
Updated comment here for setState :) I forgot to remove all the extra curlies :)